doc/classes/DisplayServer.xml

```diff
@@ -1980,21 +1996,24 @@
 		<constant name="MOUSE_MODE_MAX" value="5" enum="MouseMode">
 			Max value of the [enum MouseMode].
 		</constant>
+		<constant name="INVALID_SCREEN" value="-1">
+			The ID that refers to a nonexistent screen. This is returned by some [DisplayServer] methods if no screen matches the requested result.
```

Comment:
I would actually say this:
```suggestion
			The ID that refers to a screen that does not exist. This is returned by some [DisplayServer] methods if no screen matches the requested result.
```
But this is actually consistent with the other INVALID constants which you have likely written prior, so this is fine.